posix.mak

```diff
@@ -130,14 +108,16 @@ $(CURL_TOOLS): $(ROOT)/%: %.d
 $(TOOLS) $(DOC_TOOLS): $(ROOT)/%: %.d
 	$(DMD) $(MODEL_FLAG) $(DFLAGS) -of$(@) $(<)
 
-$(TAGS): %.tag: $(DOC)/%.html $(ROOT)/findtags
-	$(ROOT)/findtags $< > $@
+$(DOC)/chmgen : $(DOC)/chmgen.d
+	$(DMD) -g -of$@ $<
+
+ALL_OF_PHOBOS_DRUNTIME_AND_DLANG_ORG = # ???
```

Comment:
Shouldn't be a problem if he's tracking branches, as then there's no need for a history and updating the repo every time D's components get new commits.